src/quo2/components/buttons/button.cljs

```diff
@@ -124,7 +129,12 @@
                                                     :disabled colors/white-opa-5}}
            :shell            {:icon-color          colors/white
                               :label               {:style   {:color colors/white}}
-                              :background-color    {:default colors/neutral-95}}}})
+                              :background-color    {:default colors/neutral-95}}
+           :purple    {:icon-color       colors/white
+                       :label            {:style {:color colors/white}}
+                       :background-color {:default  (colors/custom-color :purple 50)
+                                          :pressed  (colors/custom-color :purple 50)
+                                          :disabled (colors/custom-color :purple 50)}}}})
```

Comment:
Hi @chrispader, thank you for your work
1. For dark it should be purple 60
2. This only covers a single case, when community theme color is purple. We need to find a better way for doing this.

Reply:
Oh ok, didn't know, that this would be "community themed". Will change the schema for token gating, so a `community-color` can be defined :)

Reply:
But the button component color would also need to be adjustable...

Reply:
Will only specific (pre-defined) "theme colors" for communities be allowed, or can any color be chosen?

Reply:
any color ([ref](https://discord.com/channels/624205794384281629/626744384565674034/1019905254134911016))

Reply:
thanks!

Reply:
Waiting for a approach from the design team on that, so this is still WIP